src/In.ProjectEKA.DefaultHip/Discovery/DiscoveryUseCase.cs

```diff
@@ -0,0 +1,24 @@
+namespace In.ProjectEKA.DefaultHip.Discovery
+{
+    using System;
+    using System.Linq;
+    using HipLibrary.Patient.Model.Response;
+
+    public static class DiscoveryUseCase
+    {
+        public static Tuple<Patient, ErrorResponse> DiscoverPatient(
+            IQueryable<Patient> patients)
+        {
+            if (!patients.Any())
+            {
+                return new Tuple<Patient, ErrorResponse>(null,
+                    new ErrorResponse(new Error(ErrorCode.NoPatientFound, "No patient found")));
+            }
+
+            if (patients.Count() <= 1) return new Tuple<Patient, ErrorResponse>(patients.First(), null);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4850723)